Tests/Unit/NetworkServerTests/ConcentratorDeduplicationTest.cs

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.Tests.Unit.NetworkServerTests
+{
+    using LoRaWan.NetworkServer;
+    using LoRaWan.NetworkServer.BasicsStation;
+    using Microsoft.Extensions.Logging.Abstractions;
+    using Moq;
+    using System;
+    using System.Threading.Tasks;
+    using Xunit;
```

Comment:
Sort so `System.*` namespaces are listed first:

```suggestion
    using System;
    using System.Threading.Tasks;
    using LoRaWan.NetworkServer;
    using LoRaWan.NetworkServer.BasicsStation;
    using Microsoft.Extensions.Logging.Abstractions;
    using Moq;
    using Xunit;
```